_includes/manuals/1.13/4.4.3_prov_templates.md

```diff
@@ -49,3 +51,36 @@ Finish templates are available for all hypervisors that support image based inst
 
 ###### User Data Templates
 User Data Templates are available for hypervisors that support customization via tools like *cloud-init*. In this case the installed machine does not need be reachable via ssh by the Foreman server. However, the installed must be able to reach Foreman or a Smart Proxy with the templates feature via http(s) to notify the setup has finished.
+
+###### PXE Loader
+
+When creating new Host, PXE Loader option must be selected in order to pass
```

Comment:
"When creating a new host, the PXE Loader option", prefer italics for names used in the UI, e.g. `_PXE Loader_`